source/blog/2017-11-06-fedora-atomic-26-to-27-upgrade.md

```diff
@@ -0,0 +1,221 @@
+---
+# Used for pa.io/blog
+title: 'Fedora 26->27 Atomic Host Upgrade Guide'
+author: dustymabe
+date: 2017-11-06
+tags: fedora, atomic
+published: true
+---
+
+
+# Introduction
+
+This week we put out the [first]()
+release of Fedora 27 Atomic Host. Some quick notes:
+
+- In Fedora 27 Atomic Host we removed kubernetes from the base OSTree.
+  See *Appendix A: Upgrading Systems with Kubernetes* for more
```

Comment:
will want to update this to cover publication of the kube migration guide the next day.